data/json/mutations/mutations.json

```
@@ -6584,5 +6584,14 @@
     "starting_trait": true,
     "valid": false,
     "cancels": [ "PROF_HELI_PILOT" ]
+  },
+  {
+    "type": "mutation",
+    "id": "FAST_REFLEXES",
+    "name": { "str": "Fast Reflexes" },
+    "points": 3,
+    "description": "You have fast reflexes allowing you to react to attacks more easily.",
```

```suggestion
    "description": "You have fast reflexes, allowing you to react to attacks more easily.",
```
maybe add something explicilty mentioning it affects dodging? "react to attacks" is kind of vague imo

---

Updated to: ```"description": "You have fast reflexes, allowing you to dodge attacks more easily."```